azure-pipelines.yml

```diff
@@ -31,7 +31,6 @@ jobs:
           versionSpec: $(PYTHON_VERSION)
           architecture: $(PYTHON_ARCH)
           addToPath: true
-        condition: eq(variables['TEST_MODE'], 'pip')
```

Comment:
`variables.TEST_MODE` doesn't exist for MNE-Connectivity, but you can see where it came from here:

https://github.com/mne-tools/mne-python/blob/b329515933915fd077495ea41de876119ac04c97/azure-pipelines.yml#L252

and where it used to get used (nowadays it's only used in the job/run title displayed in the azure UI):

https://github.com/mne-tools/mne-python/blame/04d31007afc081512146f2879937840819f00ced/azure-pipelines.yml#L210